app-frontend/src/app/components/scenes/sceneImportModal/sceneImportModal.html

```diff
@@ -24,6 +24,12 @@ <h4 class="modal-title" ng-if="$ctrl.resolve.project">
         <div class="list-group-item" ng-repeat="datasource in $ctrl.datasources.results | filter: {name: $ctrl.searchString}">
           <div class="list-group-overflow">
             <strong class="color-dark">{{datasource.name}}</strong><br>
+            {{datasource.bands.length}}
+            <ng-pluralize count="datasource.bands.length"
+                          when="{'one': 'band',
+                                'other': 'bands'
```

Comment:
you can just do `'other': '{} bands'` and delete {{datasource.bands.length}} above